install.sh

```diff
@@ -84,16 +84,25 @@ echo "================= Installing Java  ==================="
 echo "================= Installing Ruby  ==================="
 . /c7/ruby/install.sh
 
+export GIT_VERSION=2.19.1
 echo "================= Installing Git ==================="
-sudo yum install http://opensource.wandisco.com/centos/7/git/x86_64/wandisco-git-release-7-2.noarch.rpm
-sudo yum install git-2.18.0
+sudo yum install curl-devel expat-devel gettext-devel openssl-devel zlib-devel
+sudo yum install gcc perl-ExtUtils-MakeMaker
+cd /usr/src
+wget https://www.kernel.org/pub/software/scm/git/git-"$GIT_VERSION".tar.gz
+tar xzf git-"$GIT_VERSION".tar.gz
+cd git-"$GIT_VERSION"
+make prefix=/usr/local/git all
+make prefix=/usr/local/git install
```

Comment:
@ric03uec Should we really compile, or just not update it until it shows up in a package repo?

Reply:
ya. lets just wait for the official package to get updated for this. no need to build from source